src/data.js

```diff
@@ -11,6 +11,7 @@ let Task = Entity.configure({
   properties: {
     taskId:         Entity.types.SlugId,
     provisionerId:  Entity.types.String,
+    clientId:       Entity.types.String, 
```

Comment:
When modifying a data format in `data.js`, it's necessary to add a new [version](https://github.com/taskcluster/azure-entities#migrations).  This table already has a few versions, so just add one more at the end.

Reply:
ok will modify.